app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -220,24 +360,80 @@ end
         "status": 401,
         "kong_message": "No API key found in request, arr!"
     }
-    ```
+```
 
-6. Note the plugin can also be applied globally
+## More Examples
 
-   ```bash
-    $ http :8001/plugins \
-        name=exit-transformer \
-        config.handle_unknown=true \
-        config.functions=@transform.lua
+### Apply the Plugin Globally to Handle Unknown Responses
+
+Note the plugin can also be applied globally:
+
+{% navtabs %}
+{% navtab Using cURL %}
+
+```bash
+curl -X POST http://<admin-hostname>:8001/plugins/ \
+    --data "name=exit-transformer"  \
+    --data "config.handle_unknown=true" \
+    --data "config.functions=@transform.lua"
+...
+curl --header 'Host: non-existent.com' 'localhost:8000'
+```
+
+{% endnavtab %}
+{% navtab Using HTTPie %}
+
+```bash
+$ http :8001/plugins \
+    name=exit-transformer \
+    config.handle_unknown=true \
+    config.functions=@transform.lua
 
     $ http :8000 Host:non-existent.com
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Response:
+
+```bash  
     HTTP/1.1 200 OK
     ...
     X-Some-Header: some value
 
     {
         "error": true,
         "status": 404,
-        "kong_message": "no Route matched with those values, arr!"
+        "kong_message": "No Route matched with those values, arr!"
     }
-    ```
+  ```
+
+### Custom Errors by MIME Type
+
+This example shows a use case where you want custom JSON and HTML responses
+based on an [Accept header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept).
+
```

Comment:
We still need to include the code of `custom-errors-by-mimetype.lua` right? https://github.com/Kong/kong-plugin-enterprise-exit-transformer/blob/master/examples/custom-errors-by-mimetype.lua

It's a tad long though

Reply:
> We still need to include the code of `custom-errors-by-mimetype.lua` right? https://github.com/Kong/kong-plugin-enterprise-exit-transformer/blob/master/examples/custom-errors-by-mimetype.lua
> 
> It's a tad long though

@eskerda yes it is otherwise it would already be there. Since the response codes are now included, I could just have a comment line that says see codes section above?

Reply:
But then the code sample will not work.. maybe we can add it as:

```
local HTTP_MESSAGES = {
    s400 = "Bad request",
    s401 = "Unauthorized",
    -- ...
    -- See codes section above for the full list
    s511 = "Network authentication required",
    default = "The upstream server responded with %d"
}
```

or similar. Otherwise, we can always put the code sample on a github gist, or a different page and link to it?

Reply:
Added abbreviate code example as we discussed.